src/main/scala/sbtbuildinfo/ScalaCaseClassRenderer.scala

```diff
@@ -5,7 +5,7 @@ case class ScalaCaseClassRenderer(options: Seq[BuildInfoOption], pkg: String, ob
   override def extension = "scala"
 
   val traitNames = options.collect{case BuildInfoOption.Traits(ts @ _*) => ts}.flatten
-  val objTraits = if (traitNames.isEmpty) "" else " extends " ++ traitNames.mkString(" with ")
+  val objTraits = if (traitNames.isEmpty) "" else "extends " ++ traitNames.mkString(" with ")
```

Comment:
I think we should keep this as it, to avoid trailing spaces when there are no traits to mix in.

Reply:
Ah I understand this change now. (How did I misunderstand??)